### PR TITLE
Fix something went wrong error in admin section

### PR DESCRIPTION
**Pull request by dasuni-30**
### Purpose
Fix something went wrong error in admin section

### Related Issues
- None

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.extensions.v1/components/administrators/pages/administrators.tsx [hunk 1]
```diff
@@ -261,6 +262,12 @@ const CollaboratorsPage: FunctionComponent<CollaboratorsPageInterface> = (
         administratorConfig.enableAdminInvite
     );
 
+    const { data: consoleApplicationFilter } = useApplicationList(null, null, null, "name eq Console");
+
+    const consoleId: string = useMemo(() => {
+        return consoleApplicationFilter?.applications[0]?.id;
```
Comment (Achintha444) [inline features/admin.extensions.v1/components/administrators/pages/administrators.tsx:268]: remove the unecessary `return` keyword.

File: features/admin.extensions.v1/components/administrators/pages/administrators.tsx [hunk 2]
```diff
@@ -745,46 +752,49 @@ const CollaboratorsPage: FunctionComponent<CollaboratorsPageInterface> = (
...
+                    if (response?.data?.Resources.length > 0) {
+                        let adminId: string = response?.data?.Resources[0]?.id;
+
+                        if (!legacyAuthzRuntime && response?.data?.Resources?.length > 1) {
+                            const filteredRoleList: RolesV2Interface[] = response?.data?.Resources?.filter(
+                                (role: RolesV2Interface) => role?.audience?.type === RoleAudienceTypes.APPLICATION);
+
+                            if (filteredRoleList?.length > 0) {
+                                adminId = filteredRoleList[0]?.id;
+                            }
```
Comment (Achintha444) [inline features/admin.extensions.v1/components/administrators/pages/administrators.tsx:774]: adminId = filteredRoleList[0] ? filteredRoleList[0]?.id : adminId

File: features/admin.extensions.v1/components/administrators/pages/administrators.tsx (hunk 1 above)
Comment (dasuni-30) [reply features/admin.extensions.v1/components/administrators/pages/administrators.tsx]: Improved with 7da71001ffb8fb165c506dd33ee01229f6737a8f

File: features/admin.extensions.v1/components/administrators/pages/administrators.tsx (hunk 2 above)
Comment (dasuni-30) [reply features/admin.extensions.v1/components/administrators/pages/administrators.tsx:774]: I have used the shorthand of the suggestion. This suggestion doesn't cover the required logic.

**Review by Achintha444 — APPROVED** [review]

**Review by pavinduLakshan — APPROVED** [review]

**Review by brionmario — APPROVED** [review]

**Pull request merged**